cmd/root.go

```diff
@@ -36,6 +36,7 @@ var rootCmd = &cobra.Command{
 		fmt.Println("  kubefirst help")
 		progress.Progress.Quit()
 	},
+	SilenceErrors: true,
```

Comment:
... how come?

Reply:
Without this, we are doing a double print of errors, which looked pretty clunky. My understanding of this setting is that we're (aspirationally 😄) managing the printing of error messages so there was no need to tack it onto the end as well.

Reply:
Oh right, we changed the `main()` function to `Fprintln(os.Stderr, ...)`, right? Got it. Thank you!